hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -154,6 +154,14 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.datanode.netty.worker.threads</name>
+  <value>10</value>
```

Comment:
Changing defaults is an incompatible change, it should stay the same. the code is there since 2014, if someone finds it isn't a good number he can configure explicitly

Reply:
I don't have a strong opinion here. Either way works for me.